### DISCUSSION TITLE
My repository was suddenly deleted. How can I get it back?

### DISCUSSION THREAD

**Discussion by colonal** [community/community#118317] [Repositories] [answered]
How can I recover a suddenly deleted GitHub repository named "[My-Prayer](https://github.com/colonal/My-Prayer)" The last time he showed the repository was four days ago and not listed under Settings -> Repositories -> Deleted repositories.

**Comment by ItsLasa** [comment]
A deleted repository can be restored within 90 days, unless the repository was part of a fork network that is not currently empty. A fork network consists of a parent repository, the repository's forks, and forks of the repository's forks. If your repository was part of a fork network, it cannot be restored unless every other repository in the network is deleted or has been detached from the network.

read this article -https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository

**Reply by colonal** [reply]
It has not been 90 days since it was deleted. I am sure it was there last week and I did not delete it. I discovered today that it is not there.
After searching the Qasm Security log I discovered the following
The name of the repository has been changed from "My-Prayer" to "ArcaneHacks".
Then the repository was deleted
I found it in the Deleted Repositories section with the new name and I cannot recover it

There was access to my account, as a login from a different country was found. This is what the logs show.
You have taken appropriate action

Now I have to retrieve the repository. Do you know how to reach technical support?



```json
{
            "@timestamp": 1712320321204,
            "_document_id": "xxxxxxx",
            "action": "repo.rename",
            "actor": "colonal",
            "actor_id": 64502335,
            "created_at": 1712320321204,
            "old_name": "My-Prayer",
            "operation_type": "modify",
            "public_repo": true,
            "repo": "colonal/ArcaneHacks",
            "repo_id": 476071653,
            "user": "colonal",
            "user_agent": "Mozilla/5.0 (Windows NT 10.0 Win64 x64 rv:124.0) Gecko/20100101 Firefox/124.0",
            "user_id": 64502335,
            "visibility": "public"
        },
        {
            "@timestamp": 1712330150902,
            "_document_id": "xxxxxxx",
            "action": "integration_installation.repositories_removed",
            "actor": "colonal",
            "actor_id": 64502335,
            "created_at": 1712330150902,
            "integration": "Notion Workspace",
            "name": "Notion Workspace",
            "operation_type": "remove",
            "repositories_removed": [
                476071653
            ],
            "repositories_removed_names": [
                "colonal/ArcaneHacks"
            ],
            "repository_selection": "all",
            "topic": "github.repositories.v1.Deleted",
            "user": "colonal",
            "user_id": 64502335
        },
        {
            "@timestamp": 1712330150817,
            "_document_id": "xxxxxxx",
            "action": "integration_installation.repositories_removed",
            "actor": "colonal",
            "actor_id": 64502335,
            "created_at": 1712330150817,
            "integration": "deepsource.io",
            "name": "deepsource.io",
            "operation_type": "remove",
            "repositories_removed": [
                476071653
            ],
            "repositories_removed_names": [
                "colonal/ArcaneHacks"
            ],
            "repository_selection": "all",
            "topic": "github.repositories.v1.Deleted",
            "user": "colonal",
            "user_id": 64502335
        }
```

![Screenshot 2024-04-08 201127](https://github.com/community/community/assets/64502335/194d512e-5fa6-4a76-ad33-8f029be04cbb)

**Reply by ItsLasa** [reply, marked as answer]
Contact GitHub Support...I think it's best way to recover your deleted repo

**Reply by v1dya-sagar** [reply]
You can Contact Github Support [here](https://support.github.com/request)

**Comment by U7P4L-IN** [comment]
how to restore github deleted repository if don't showing restoring option?

![Screenshot_2025-01-12-07-05-39-08_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/7317347d-fae6-4c07-936a-5463d92bccb1)

**Reply by colonal** [reply]
Contact GitHub Support, it's the best way to recover your deleted repo
https://support.github.com/request/restore-data?flow=repository_restore

**Reply by U7P4L-IN** [reply]
But bot can't help me yet🥲
